### PR TITLE
Update seal docs to reflect 1.3 changes.

### PR DESCRIPTION
**Pull request by ncabatoff**

### CODE REVIEW

File: website/pages/docs/concepts/seal.mdx [hunk 1]
```diff
@@ -49,18 +56,19 @@ This process is stateful: each key can be entered via multiple mechanisms
...
 key to be on a distinct machine for better security.
 
-Once a Vault is unsealed, it remains unsealed until one of two things happens:
+Once a Vault node is unsealed, it remains unsealed until one of two things happens:
 
 1. It is resealed via the API (see below).
 
 2. The server is restarted.
 
+3. Something sufficiently bad happens at the storage layer that Vault decides
```
Comment (mjarmy) [inline website/pages/docs/concepts/seal.mdx:65]: Maybe a slightly less colloquial wording here would be better.

Comment (ncabatoff) [reply website/pages/docs/concepts/seal.mdx:65]: "Vault's storage layer encounters an unrecoverable error"?

Comment (mjarmy) [reply website/pages/docs/concepts/seal.mdx:65]: yep, sounds good

**Comment by ncabatoff** [comment]
Ok images have been added.  I feel bad having my crapping drawings in our nice website, but I guess an imperfect diagram is better than none.

**Pull request merged**